tests/semantics/manifest.ttl

```diff
@@ -1,12 +1,30 @@
-PREFIX rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
-PREFIX rdfs:   <http://www.w3.org/2000/01/rdf-schema#>
-PREFIX mf:     <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#>
-PREFIX test:   <http://www.w3.org/2001/sw/DataAccess/tests/test-dawg#>
-PREFIX xsd:    <http://www.w3.org/2001/XMLSchema#>
-PREFIX trs:    <https://w3c.github.io/rdf-star/tests/semantics#>
+## Distributed under both the "W3C Test Suite License" [1]
+## and the "W3C 3-clause BSD License".
+## [1] https://www.w3.org/Consortium/Legal/2008/04-testsuite-license
+## [2] https://www.w3.org/Consortium/Legal/2008/03-bsd-license
+
+@PREFIX rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@PREFIX rdfs:   <http://www.w3.org/2000/01/rdf-schema#> .
+@PREFIX mf:     <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#> .
+@PREFIX rdft:   <http://www.w3.org/ns/rdftest#> .
+@PREFIX trs:    <https://w3c.github.io/rdf-star/tests#> .
+@PREFIX dct: <http://purl.org/dc/terms/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+
 
 trs:manifest a mf:Manifest;
-  rdfs:label "RDF-star Semantics tests";
+  rdfs:label "RDF-star Semantics tests"@en ;
+  rdfs:label "La suite des tests pour la sémantique de RDF-star"@fr;
+	rdfs:label "Conjunto de pruebas para la semántica de RDF-star"@es;
```

Comment:
Alignment error.

Please use spaces not tabs.

Here, your tabs are not the same a GH tabs.

Reply:
I'm not sure to understand, but I'll find how to solve this. Thanks.

Reply:
solved